apps/mobile/src/utils/feature-flag.ts

```diff
@@ -1,3 +1,57 @@
+import { useEffect } from 'react';
+import { Platform } from 'react-native';
+
+import { AutoEnvAttributes, ReactNativeLDClient } from '@launchdarkly/react-native-client-sdk';
+import * as Application from 'expo-application';
+
+function getDeviceId() {
+  if (Platform.OS === 'android') {
+    return Promise.resolve(Application.getAndroidId());
+  } else {
+    return Application.getIosIdForVendorAsync();
+  }
+}
+
+// TODO: this is used for localization and it should be revised in favor
```

Comment:
Maybe we can `TODO` this now? I added it to hide some features from internal release but it could be updated now to use launchdarkly

Reply:
We use it in other places too, to hide settings and other things based on `ENV` so they aren't shown to leatherhood users